src/core.js

```diff
@@ -441,7 +441,6 @@ $.extend( $.validator, {
 			if ( errors ) {
 				// add items to error list and map
 				$.extend( this.errorMap, errors );
-				this.errorList = [];
```

Comment:
I also dont get why we have a `errorMap` and a `errorList`. Those are event handed over to library consumers in `validator.settings.showErrors` callbacks, see http://jqueryvalidation.org/validate/#showerrors